gatsby/src/components/container/container.tsx

```diff
@@ -13,11 +15,10 @@ const Container: React.FC<IProps> = ({
 }) => {
   return (
     <div
-      className={classNames(
-        { container: variant === 'normal' },
-        { 'container-fluid': variant === 'fluid' },
-        className,
-      )}
+      className={classNames(className, {
+        [styles.container]: variant === 'normal',
+        [styles.container__fluid]: variant === 'fluid',
+      })}
```

Comment:
Je potřeba dělat moduly, když to máme stejně naimportovaný globálně? Nebude to akorát dělat bordel?

Reply:
Nelíbí se mi moc hrabání do stylů toho design systému, to udělá ještě větší bordel. A přetěžovat je nemá smysl, protože bych přepisoval snad všechno, co tam pro ten container je.

Reply:
Dobře no, ale proč přepisovat defaultní container, když ho "prostě teď potřebuješ jen ty"?

Reply:
Protože všude na té stránce je to odsazení stejný?

Reply:
Původní container dělá odsazení sám za sebe. Tak místo `margin` zkus používat jen `margin-top` a `margin-bottom`

Reply:
Vždyť jsem ve u content-boxu `margin` předělal tak, že není zleva ani zprava. Ale to pořád neřeší problém, že 30px zleva a zprava, které má `container` z design systému, je o 10px víc, než je v návrhu.